### PR TITLE
Bugfix: correct copy vs copyTextWithoutNewlines behavior

### LINKED ISSUE #3782: Keybindings "copy" and "copyTextWithoutNewlines" seem to have opposite behaviors
# Environment

```none
Windows build number: 19033.1
Windows Terminal version (if applicable): 0.7.3291.0

Any other software? WSL
```

# Steps to reproduce

1. Set keybindings as following:
```json
    "keybindings": [
        { "command": "copy", "keys": [ "ctrl+c" ] },
        { "command": "copyTextWithoutNewlines", "keys": [ "ctrl+shift+c" ] }
]
```

2. copy multiple lines of text from terminal with "ctrl+c", paste to notepad
3. copy multiple lines of text from terminal with "ctrl+shift+c", paste to notepad

# Expected behavior

Step 2 should paste texts with line breaks
Step 3 should paste texts without line breaks (one line with a lot of spaces)

# Actual behavior
Step 2 results in one line of texts
Step 3 results in multiple line of text

### PR DESCRIPTION
**Pull request by carlos-zamora**
## Summary of the Pull Request
Corrected behavior for (de)serialization. Changed variable name for clarification.

## References
N/A

## PR Checklist
* [ ] Closes #3782
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Detailed Description of the Pull Request / Additional comments
This was on me. Should've caught it in the code review. Sorry!

## Validation Steps Performed
Tests
And personal tests

### CODE REVIEW

File: doc/cascadia/profiles.schema.json [hunk 1]
```diff
@@ -108,8 +108,8 @@
             "action": { "type": "string", "pattern": "copy" },
             "trimWhitespace": {
```
Comment (DHowett-MSFT) [inline doc/cascadia/profiles.schema.json]: the name changed

File: src/cascadia/LocalTests_TerminalApp/KeyBindingsTests.cpp [hunk 2]
```diff
@@ -214,24 +214,24 @@ namespace TerminalAppLocalTests
 
         {
             Log::Comment(NoThrowString().Format(
-                L"Verify that `copy` without args parses as Copy(TrimWhitespace=false)"));
+                L"Verify that `copy` without args parses as Copy(StripNewlines=true)"));
```
Comment (DHowett-MSFT) [inline src/cascadia/LocalTests_TerminalApp/KeyBindingsTests.cpp]: this is backwards

File: src/cascadia/LocalTests_TerminalApp/KeyBindingsTests.cpp [hunk 3]
```diff
@@ -214,24 +214,24 @@ namespace TerminalAppLocalTests
...
             VERIFY_IS_NOT_NULL(realArgs);
             // Verify the args have the expected value
-            VERIFY_IS_FALSE(realArgs.TrimWhitespace());
+            VERIFY_IS_FALSE(realArgs.StripNewlines());
         }
 
         {
             Log::Comment(NoThrowString().Format(
-                L"Verify that `copyTextWithoutNewlines` parses as Copy(TrimWhitespace=true)"));
+                L"Verify that `copyTextWithoutNewlines` parses as Copy(StripNewlines=false)"));
```
Comment (DHowett-MSFT) [inline src/cascadia/LocalTests_TerminalApp/KeyBindingsTests.cpp]: sdrawkcab si siht

File: src/cascadia/LocalTests_TerminalApp/KeyBindingsTests.cpp (hunk 2 above)
Comment (DHowett-MSFT) [reply src/cascadia/LocalTests_TerminalApp/KeyBindingsTests.cpp]: the whole test

File: src/cascadia/TerminalControl/TermControl.cpp [hunk 4]
```diff
@@ -919,7 +919,7 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
             if (_terminal->IsCopyOnSelectActive())
             {
-                CopySelectionToClipboard(!shiftEnabled);
+                CopySelectionToClipboard(shiftEnabled);
```
Comment (DHowett-MSFT) [inline src/cascadia/TerminalControl/TermControl.cpp]: how did this get backwards anyway?

Comment (DHowett-MSFT) [reply src/cascadia/TerminalControl/TermControl.cpp]: So what happened here was that the keybinding deserializer for `copy` was creating a `copy, trimWhitespace=false`?

**Comment by carlos-zamora** [comment]
Abandoning this PR. Check out #3823 for the patch. Will probably revive this PR when we decide on how to move forward. Issue to track this is #3824

**Pull request closed without merging**